source/style-guide/seo-guidelines.txt

```diff
@@ -0,0 +1,84 @@
+.. _seo-guidelines:
+
+=====================================
+Search Engine Optimization Guidelines
+=====================================
+
+.. default-domain:: mongodb
+
+Search Engine Optimization (SEO) considers the actual terms that users 
+enter into serach engines (keywords) and employs best practices to 
+improve traffic to web pages from search engines. The following 
+guidance should help you optimize your documentation pages for search.
+
+Titles
+------
+
+Title pages and subsections according to the following SEO best 
+practices:
+
+- Use a maximum of 70 characters.
+
+- Include target keywords.
+
+- Avoid excessive or irrelevant words (keyword stuffing).
+
+- Use a unique title for every page.
+  
+To learn more, see :ref:`titles-and-headings`.
+
+Alternative Text
+----------------
+
+Screen readers read alternative text aloud so that users can better
+understand an on-page image. Specify alterative text according to the 
+following SEO best practices:
+
+- Use a maximum of 125 characters per image.
+
+- Describe the image with specific language.
+
+- If it makes sense, include keywords.
+
+To learn more, see :ref:`write-for-accessibility`.
+
+Keywords
+--------
+
+You can add the actual terms that users enter into search engines 
+(keywords) to your page to improve its SEO. Add keywords according to 
+the following best practices:
```

Comment:
Suggestion:

I think it might be better to suggest that keywords may not help SEO nor be used to match up with search terms, at least with Google. See
https://developers.google.com/search/docs/crawling-indexing/special-tags#unsupported
for more info.

Reply:
Yes, there's two parts to these recommendations. We should use the language that users search for in our content. But, search engines might not account for the keyword tags that we use.